package.json

```diff
@@ -7,7 +7,7 @@
     "automated testing",
     "android"
   ],
-  "version": "1.2.55",
+  "version": "1.2.58",
```

Comment:
Yeah skipped intentionally. Since versions 1.2.56 and 1.2.57 are having minor issue.